### PR TITLE
Chapter3/homework/3

### PR DESCRIPTION
**Pull request by jeanroyce**
Not much to say, done with small changes :)

### CODE REVIEW

**Comment by jeanroyce** [comment]
Just in case,  I made Null or Empty validations in public methods and strings in ctor.

**Comment by Almantask** [comment]
> Just in case, I made Null or Empty validations in public methods and strings in ctor.

Which is a very good thing 😃

File: Src/BootCamp.Chapter/Equipment.cs [hunk 1]
```diff
@@ -2,90 +2,111 @@
...
-        }
-
-        private Shoulderpiece _leftShoulder;
-        public void SetLeftShoulder(Shoulderpiece should)
-        {
-
+            get
+            {
+                UpdateDefense();
+                return _weight;
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Equipment.cs:35]: Weight is not defense.

File: Src/BootCamp.Chapter/Item.cs [hunk 2]
```diff
@@ -1,26 +1,19 @@
...
+        public float Weight { get; }
+        
         public Item(string name, decimal price, float weight)
         {
-            _name = name;
-            _price = price;
-            _weight = weight;
+            if (string.IsNullOrEmpty(name)) throw new ArgumentNullException(nameof(name));
+            Name = name;
+            Price = price;
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Item.cs:15]: Could have done the same for price and weight (cannot be negative or 0)

**Pull request closed without merging**